include/boost/geometry/algorithms/detail/buffer/buffer_policies.hpp

```diff
@@ -127,23 +127,23 @@ struct buffer_turn_info
     robust_point_type rob_pi, rob_pj, rob_qi, rob_qj;
 #endif
 
-    int count_within;
+    signed_index_type count_within;
```

Comment:
counts are not indexes.
Can you make them std::size_t, besides count_in_original which should stay int?